### ISSUE TITLE
The taskbar does not auto-hide anymore.

### ISSUE THREAD

**Issue by dertuxmalwieder** [cairoshell/cairoshell#512]
**Describe the bug**
On Windows 10 with Cairo 0.3.7174.0, I cannot auto-hide the taskbar anymore.

**Steps to reproduce**
1. Enable the option to auto-hide the taskbar,
2. Find that the taskbar refuses to disappear.

**Expected behavior**
The taskbar should disappear automatically.

**System information:**
 - Windows version: 10
 - Cairo version: 0.3.7174.0
 - 32-bit or 64-bit: 64-bit
 - In Cairo Settings > Advanced, is Cairo set as shell?: No(t yet).

**Screenshots**
![oOgs6Qe](https://user-images.githubusercontent.com/125629/103311720-62f92680-4a1b-11eb-89ce-34c102b0c00f.png)

**Comment by dremin** [comment]
Does the auto-hide work again if you restart Cairo? Sometimes it can stop working when changing the other taskbar settings.

**Comment by dertuxmalwieder** [comment]
It does indeed. Can I help to debug the problem? I think the last action I performed was switching between "taskbar overlaps windows" and auto-hiding.

**Comment by dremin** [comment]
I believe the size and position settings are what typically cause the issue to manifest. Did you adjust any of those settings?

**Comment by dertuxmalwieder** [comment]
Yes, I enabled the full display width and multi-monitor settings. The position is just fine out of the box.
